cmd/provider/main.go

```diff
@@ -39,6 +40,7 @@ func main() {
 		syncInterval   = app.Flag("sync", "Sync interval controls how often all resources will be double checked for drift.").Short('s').Default("1h").Duration()
 		pollInterval   = app.Flag("poll", "Poll interval controls how often an individual resource should be checked for drift.").Default("1m").Duration()
 		leaderElection = app.Flag("leader-election", "Use leader election for the conroller manager.").Short('l').Default("false").OverrideDefaultFromEnvar("LEADER_ELECTION").Bool()
+		ratelimiterRPS = app.Flag("ratelimiter-rps", "Rate limiter RPS controls the requeues per second defined for the global ratelimiter.").Default(strconv.Itoa(ratelimiter.DefaultGlobalRPS)).Int()
```

Comment:
Would the [new flag](https://github.com/crossplane/provider-aws/blob/29edbd1/cmd/provider/main.go#L43) merged with https://github.com/crossplane/provider-aws/pull/1187 work for you?

Reply:
@muvaf Oh fantastic, I wasn't aware there was already an open PR that addressed this! Will close, thank you 👍